packages/components/icon/examples/IconAsPropExample.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import { Icon } from '@contentful/f36-components';
+import { MdAccessAlarm } from 'react-icons/md';
+
+export default function IconAsPropExample() {
+  return <Icon as={MdAccessAlarm} variant="secondary" />;
+}
```

Comment:
Unfortunately because we remove the imports, this example is not very clear in the page

![Screenshot 2022-01-14 at 16 07 25](https://user-images.githubusercontent.com/6597467/149537891-a770d3a1-af00-4004-a7d5-516b105cf72b.png)

I only understood where the `MdAccessAlarm` was coming from after I read the code in the PR

I don't know how to solve this tbh, maybe a comment in the example? maybe a SVG instead of `react-icons`?

🤷

Reply:
there is an example with SVG icons. I will add a comment